package.json

```
@@ -31,6 +31,7 @@
   "scripts": {
     "clean": "FORCE_COLOR=1 turbo run clean && rm -rf ./node_modules",
     "format": "prettier . --write",
+    "lint": "turbo run lint",
```

for the time being, we need to prepend turbo scripts with `FORCE_COLOR=1` to preserve underlying color. hoping they fix this soon

```suggestion
    "lint": "FORCE_COLOR=1 turbo run lint",
```